test/goalc/goalc_test_list.cpp

```diff
@@ -0,0 +1,18 @@
+// this list should contain all of the goalc test files.
```

Comment:
@xTVaser I combined all of your test files into a single giant file.  This seems to make a pretty big different because it ends up parsing the inja header only once.  Let me know if think this could have some negative consequence.